package.json

```diff
@@ -19,9 +19,12 @@
         "dev": "nx run dev --project=${0}",
         "docs": "nx run docs:build",
         "docs:serve": "nx run docs:serve",
-        "bump": "yarn workspaces foreach --no-private version"
+        "bump": "yarn workspaces foreach --no-private version",
+        "changeset:add": "changeset"
     },
     "devDependencies": {
+        "@changesets/changelog-github": "^0.4.8",
```

Comment:
Do we need this one? 

Looks like it is used by the github action (but the action should use it itself) 

Might cut down on the added packages to remove it.

Reply:
deleted

Reply:
We've added `@changesets/changelog-github` to our [changeset configuration file](https://github.com/EventStore/Design-System/blob/97aa643ff0701a6eabf019ef7b4da3aa2fce01f0/.changeset/config.json#L3).

This package is used by our GitHub Actions workflow to create well-formatted changelogs for each new release.
Therefore, we need to have it installed as a development dependency in our project.

Reply:
![image](https://github.com/EventStore/Design-System/assets/25894545/7e8e7c57-9c2b-43fd-abf5-6ef6b2754004)
The error when the devDependency is missing.